apps/api/src/controllers/users.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-process-env */
+/* eslint-disable eqeqeq */
+/* eslint-disable consistent-return */
+/* eslint-disable require-await */
+/* eslint-disable no-magic-numbers */
+/* eslint-disable max-len */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-mixed-requires */
+/* eslint-disable global-require */
+module.exports = app => {
+    require('dotenv-safe').config();
+    const jwt = require('jsonwebtoken');
+    const { models, connectDb } = require('../../config/db'),
+     controller = {};
+
+    /* Autenticação login */
+    controller.login = async (req, res) => {
+        connectDb().then(async() => {
+            const { email, password } = req.body,
+             user = await models.User.findOne({ email }).select('+password');
+
+            if (!user) {
+                return res.status(400).send({ 'error': 'Usuário não encontrado' });
+            }
+
+            if (password !== user.password) {
+ return res.status(400).send({ 'error': 'Senha inválida' });
+}
```

Comment:
Não poderia adicionar o password junto do email no findOne, e em seguida apenas verificar se o usuário foi encontrado ou não?
```suggestion
             user = await models.User.findOne({ email: email, password: password });

            if (!user) {
                return res.status(400).send({ 'error': 'E-mail ou senha inválida!' });
            }
```

Reply:
Feito, esse select('+password') tinha inserido quando minha função estava de outra maneira, havia esquecido de alterar